mslib/mscolab/file_manager.py

```diff
@@ -279,7 +292,7 @@ def get_change_content(self, ch_id):
         return change_content
 
     def set_version_name(self, ch_id, p_id, u_id, version_name):
-        if not self.is_admin(u_id, p_id):
+        if not self.is_admin(u_id, p_id) and self.is_collaborator(u_id, p_id):
```

Comment:
you probably want to set this

if not (self.is_admin(u_id, p_id) or self.is_collaborator(u_id, p_id)):